source/Mlos.Proto/GeneratePythonGrpcCode.cmd

```diff
@@ -4,6 +4,8 @@
 @echo off
 rem We need to recreate the structure of the python module in the input to grpc
 rem to have the imports be generated correctly. See https://github.com/grpc/grpc/issues/9575#issuecomment-293934506
+rem Note: We currently do not use imports in order to keep OptimizerMonitoringService
```

Comment:
The comments in lines 5-6 refer to imports in .py files. You may wish to disambiguate and say that we don't use includes in the .proto files.

Reply:
[zaporter/grpc 31cc56f5] Adam's suggestions